p2p/errors.go

```diff
@@ -189,3 +197,131 @@ type ErrCurrentlyDialingOrExistingAddress struct {
 func (e ErrCurrentlyDialingOrExistingAddress) Error() string {
 	return fmt.Sprintf("connection with %s has been established or dialed", e.Addr)
 }
+
+type ErrInvalidIP struct {
```

Comment:
I would strongly simplify this by re-using the existing `ErrNetAddressInvalid`.

Reply:
Updated.